src/main/java/burp/ServerLibrary.java

```diff
@@ -7,7 +7,7 @@
 public interface ServerLibrary extends Library {
     ServerLibrary INSTANCE = Native.load((Platform.isMac() ? "lib" : "") +  "server." + (Platform.isWindows() ? "dll" : Platform.isMac() ? "dylib" : "so"), ServerLibrary.class);
 
-    String StartServer(String address);
+    String StartServer(String interceptProxyAddress, String burpProxyAddress, String emulateProxyAddress);
```

Comment:
Perhaps add an options struct and provide that to the function instead, 3 parameters are beginning to become quite long and hard to remember the correct order.